HIP Documentation/HIP Programming Guide.rst

```diff
@@ -0,0 +1,23 @@
+=========================
+HIP Programing Guide
+=========================
+
+What is this repository for?
+
+HIP allows developers to convert CUDA code to portable C++. The same source code can be compiled to run on NVIDIA or AMD GPUs. 
+
+Key features 
+==============
+
+HIP is very thin and has little or no performance impact over coding directly in CUDA or hcc “HC” mode.
+
+HIP allows coding in a single-source C++ programming language including features such as templates, C++11 lambdas, classes, namespaces, 
+and more.
+
+HIP allows developers to use the “best” development environment and tools on each target platform.
+
+The “hipify” tool automatically converts source from CUDA to HIP.
+
+Developers can specialize for the platform (CUDA or hcc) to tune for performance or handle tricky cases
```

Comment:
(CUDA or hcc) -> (CUDA or HIP)

The new HIP Clang is supporting HIP as a first class language. It no longer is doing so through the HCC language.